pkg/cmd/repo/edit/edit.go

```diff
@@ -403,16 +408,26 @@ func interactiveRepoEdit(opts *EditOptions, r *api.Repository) error {
 				return err
 			}
 		case optionVisibility:
-			opts.Edits.Visibility = &r.Visibility
-			//nolint:staticcheck // SA1019: prompt.SurveyAskOne is deprecated: use Prompter
-			err = prompt.SurveyAskOne(&survey.Select{
-				Message: "Visibility",
-				Options: []string{"public", "private", "internal"},
-				Default: strings.ToLower(r.Visibility),
-			}, opts.Edits.Visibility)
+			opts.Edits.Visibility = new(string)
+			*opts.Edits.Visibility = r.Visibility
```

Comment:
I think it's generally safer to change the address pointed to by `opts.Edit.Visibility` (as it was before) than changing the value at the address.
```suggestion
			opts.Edits.Visibility = &r.Visibility
```